stdlib/universe/filter.go

```diff
@@ -359,3 +360,40 @@ func (RemoveTrivialFilterRule) Rewrite(ctx context.Context, filterNode plan.Node
 	anyNode := filterNode.Predecessors()[0]
 	return anyNode, true, nil
 }
+
+// MergeFiltersRule merges Filter nodes whose body is a single return to create one Filter node.
+type MergeFiltersRule struct{}
+
+func (MergeFiltersRule) Name() string {
+	return "MergeFiltersRule"
+}
+
+// need to think of pattern
```

Comment:
I think that this comment and the one below at line 376 aren't needed now.

Reply:
Let me make that change!